src/ports/postgres/modules/deep_learning/madlib_keras_helper.py_in

```diff
@@ -18,6 +18,7 @@
 # under the License.
 
 import numpy as np
+import plpy
```

Comment:
We are not using `plpy` here, may be we should add it when we have use for it.

Reply:
done